packages/xo-web/src/common/report-bug-button.js

```diff
@@ -53,26 +74,25 @@ const reportOnSupportPanel = async ({
     formData.append('attachments', content, name)
   })
 
-  await Promise.all([
-    timeout.call(xoaUpdater.getLocalManifest(), 5e3).then(
-      manifest =>
-        formData.append(
-          'attachments',
-          createBlobFromString(JSON.stringify(manifest, null, 2)),
-          'manifest.json'
-        ),
-      error => logger.warn('cannot get the local manifest', { error })
-    ),
-    timeout.call(checkXoa(), 5e3).then(
-      xoaCheck =>
-        formData.append(
-          'attachments',
-          createBlobFromString(stripAnsi(xoaCheck)),
-          'xoaCheck.txt'
-        ),
-      error => logger.warn('cannot get the xoa check', { error })
-    ),
-  ])
+  await Promise.all(
+    ADDITIONAL_FILES.map(
+      ({
+        fetch,
+        format = identity,
+        name,
+        timeout: timeout_ = DEFAULT_TIMEOUT,
```

Comment:
Not necessary, inline.

Reply:
Can you please tell me where i can inline it?

Reply:
Simply make it the same for all files.